docs/directive-validation.md

```diff
@@ -0,0 +1,344 @@
+# Directive validation
+
+By default, Mercurius validation supports `@constraint` Directives out of the box. It is defined as follows:
```

Comment:
Is there a way to disable this?

Reply:
Not at the moment, I can add an option to disable directive validation for sure! The total options would look like?

```js
app.register(mercuriusValidation, {
  mode: 'JTD' | 'JSONSchema', // Default 'JSONSchema'
  directiveValidation: false, // Default true
  schema: {
	...
  }
})
```

Reply:
Yes exactly!